Source/WebGPU/Configurations/WebGPU.xcconfig

```diff
@@ -143,3 +143,8 @@ EXCLUDED_SOURCE_FILE_NAMES_ = *.swift;
 
 BUILD_LIBRARY_FOR_DISTRIBUTION = YES; // This is required for all frameworks in the OS.
 RUN_SWIFT_ABI_CHECKER_TOOL = NO
+
+// When back-deploying for STP and downlevel builds, force Xcode to embed any
+// compatibility runtimes alongside WebKit.framework.
+ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = $(USE_STAGING_INSTALL_PATH);
```

Comment:
Have you actually verified this fixes backdeployed STP builds? As I understand it, we don't expect `USE_STAGING_INSTALL_PATH` to be set for STP, only for downlevel updates.

At a quick glance, maybe 'WK_USE_OVERRIDE_FRAMEWORKS_DIR' would be better here, since that covers both STP and downlevel.